### ISSUE TITLE
advanced helper panel failing

### ISSUE THREAD

**Issue by ferib** [ferib/lua-obfuscator-plugin#6]
![image](https://github.com/ferib/lua-obfuscator-plugin/assets/22753171/6a2be34d-9d99-44a6-ae2b-eec486cec845)

This happened on 2.1.0. master. Not sure if I ducked something or if unix/windows pathing is cucked 🤔

**Comment by ferib** [comment]
seems like the `webView.html` is located at `./dist/webView.html` and not `./dist/utils/webView.html` 🤔 

```js
export function getWebviewContent() {
  return fs.readFileSync(__dirname + "\\..\\webView.html", "utf8");
}
```

**Comment by ferib** [comment]
Fixed in #10  ?
